tests/ImageSharp.Tests/Formats/GeneralFormatTests.cs

```diff
@@ -111,26 +111,27 @@ public void ImageCanConvertFormat()
 
             foreach (TestFile file in Files)
             {
-                Image image = file.CreateImage();
-
-                using (FileStream output = File.OpenWrite($"{path}/{file.FileNameWithoutExtension}.gif"))
+                using (Image image = file.CreateImage())
                 {
-                    image.SaveAsGif(output);
-                }
+                    using (FileStream output = File.OpenWrite($"{path}/{file.FileNameWithoutExtension}.bmp"))
```

Comment:
ok I think I've figured out how to fix the intellisense in VS.

1. close VS
2. delete `.vs` folder
3. run `dotnet restore` in the root repo folder
4. restart VS 

You should now see the file path based APIs, it looks like VS is caching the reference to the netstandard1.1 output instead of moving to the netstandard1.3 one it should now be using.